packages/flame/lib/src/camera/camera_component.dart

```diff
@@ -278,4 +278,9 @@ class CameraComponent extends Component {
       boundedBehavior.bounds = bounds;
     }
   }
+
+  /// Returns true if this camera is able to see the [component].
+  bool sees(PositionComponent component) {
```

Comment:
I think `isInView` was clearer

Reply:
It would have been clearer if it was a Component's method:
```
if (component.isInView(camera)) { ... }
```
However, when it is a method on camera, then the statement reads as *"If camera is in view of component ..."*, which is confusing.

Reply:
Makes sense. Hmm... I don't think `sees` is super clear either, maybe `canSee`? Let me think until tomorrow (maybe you'll come up with some other alternatives too).

Reply:
Well today, but when I wake up 😅

Reply:
`canSee` is not bad, I like it

Reply:
Fixed it to `canSee`, jeez naming things is hard :D